community/kernel/src/test/java/org/neo4j/kernel/impl/api/index/IndexPopulationJobTest.java

```diff
@@ -492,7 +492,7 @@ public void process( IndexEntryUpdate<?> update ) throws IOException, IndexEntry
                             added.add( Pair.of( update.getEntityId(), update.values()[0].asObjectCopy() ) );
                             break;
                         default:
-                            throw new IllegalArgumentException( update.updateMode().name() );
+                            // ignore removals
```

Comment:
do not think that's a right way of ignoring removals, instead add an additional case for removals and keep default case that throw exception

Reply:
This is in a test and the test asserts what the populator sees, but I can make this change anyway.